networkx/algorithms/d_separation.py

```diff
@@ -7,6 +7,19 @@
 to the actual distribution parameters.  See [1]_ for a formal
 definition.
 
+To understand d-separation intuitively, consider the following example:
+
+For a pair of two nodes, u and v, all paths are considered open initially, unless
+they contain a collider along the path. d-separation is concerned with blocking all
+paths between these u and v. A path is considered blocked by the
+separating set if all open paths contain some variable in the d-separating
+set that is not a collider, or the descendant of a collider.
```

Comment:
I'm not familiar with this terminology. Can you make it clear that "open" is a label being defined by this operation and not a technical term the reader has to look up?  Also, the term "collider" is not well established yet.  Maybe put the paragraph that defines it, or at least before you use the term here. 

In the paragraph about the collider, maybe make it more clear what "open" means by saying:
`then the path through the collider is called "open".` But maybe that is an incorrect definition of open -- I am only trying to guess at what open refers to here. We have no topology right?

Reply:
Okay, I tried explaining it a bit more. The definition of d-separation in words is quite involved because you have to define what an "open", or "blocked" path is. WDYT of the new writeup?